### DISCUSSION TITLE
What are the advantages?

### DISCUSSION THREAD

**Discussion by parsisoftware** [pyg-team/pytorch-frame#260] [Q&A]
Hello,
Would you please tell me what is the advantages of using this library in comparison with using available encodings for different column types in the dataset?

**Comment by weihua916** [comment]
Good question :) Major advantages are 
- handling more column types (text, multi-categorical, embedding, timestamp etc)
- better modeling via SOTA tabular deep learning models.
- Just easier to use --- stitching together existing encodings for heterogeneous columns is often tedious.
